src/index.ts

```diff
@@ -104,104 +104,98 @@ export const forceH264Sdp = (sdp: any) => {
  * @param opts browser view options
  */
 export const createWindow = (url: string, opts?: BrowserWindowConstructorOptions) => {
-    return new Promise<BrowserWindow>((resolve, reject) => {
-        try {
-            const bw = new BrowserWindow({
-                ...opts,
-                show: false,
-            });
-
-            bw.on("page-title-updated", (e) => {
-                if (opts && opts.title) {
-                    e.preventDefault();
-                }
-            });
-
-            bw.once("ready-to-show", () => {
-                bw.show();
-                resolve(bw);
-            });
-
-            bw.loadURL(url);
-        } catch (err) {
-            reject(err);
+  return new Promise<BrowserWindow>((resolve, reject) => {
+    try {
+      const bw = new BrowserWindow({
+        ...opts,
+        show: false,
+      });
+
+      bw.on("page-title-updated", (e) => {
+        if (opts && opts.title) {
+          e.preventDefault();
         }
-    });
-};
+      });
 
-/**
- * {createStream} options object type
- */
-export interface ICreatStreamOptions {
-    audio: boolean;
-    minWidth: number;
-    maxWidth: number;
-    minHeight: number;
-    maxHeight: number;
-}
+      bw.once("ready-to-show", () => {
+        bw.show();
+        resolve(bw);
+      });
+
+      bw.loadURL(url);
+    } catch (err) {
+      reject(err);
+    }
+  });
+};
 
 /**
  * Create a stream
  * @param source stream source
  * @param opts stream options
  */
-export const createStream = (source: DesktopCapturerSource, opts?: ICreatStreamOptions) => {
-    return navigator.mediaDevices.getUserMedia({
-        audio: (opts || {} as any).audio || false,
-        video: {
-          mandatory: {
-              ...opts,
-            chromeMediaSource: "desktop",
-            chromeMediaSourceId: source.id,
-          },
-        },
-      } as any);
+export const createStream = (source: DesktopCapturerSource, opts?: MediaStreamConstraints) => {
+  return navigator.mediaDevices.getUserMedia({
+    ...opts,
+    audio: {
+      mandatory: {
+        chromeMediaSource: "system",
```

Comment:
Why would we use "system" instead of "desktop" here?

Reply:
I don't remember why, and can't find the context at this time 😞 

We can switch to "desktop" and test/maybe change, though we know this to be working currently...